### PR TITLE
Android article

### PR DESCRIPTION
**Pull request by pya35**

### CODE REVIEW

File: src/fr/articles/how-to-test-android.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+### Présentation des principales options
+- **TalkBack** : c'est le lecteur d'écran pour tous les devices Android. Il permet de vocaliser toutes les informations (utiles) présentes à l'écran. Outil indispensable pour les non-voyants et mal-voyants.
+- **Navigation clavier** : permet d'utiliser son téléphone à partir d'un clavier externe (généralement en bluetooth), utile pour les personnes ayant des difficultés à utiliser l'écran tactile.
+- **Voice Access** : option pour piloter son téléphone entièrement à la voix. Indispensable pour les personnes ne pouvant pas interagir physiquement avec le device ou avec un contacteur externe.
+- **Taille de police** : permet d'ajuster la taille des polices de caractères dans les applications qui gèrent cette option.
+
+
+### Paramétrage des raccourcis d'accessibilité
+En premier lieu, il est nécessaire d'installer certains outils depuis le Play Store Google:
```
Comment (MewenLeHo) [inline src/fr/articles/how-to-test-android.md:21]: Google :

File: src/fr/articles/how-to-test-android.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+
+3. La navigation clavier permet également de vérifier que l'ordre de navigation est respecté.
+
+
+### Voice Access
+Lorsque « Voice Access » est activé, vous pouvez énoncer des commandes comme :
+- « Retourner à l’écran d’accueil »
+- « Appuyer sur *nom de l’élément* »
+- « Ouvrir *nom de l’app* »
```
Comment (MewenLeHo) [inline src/fr/articles/how-to-test-android.md:68]:
```suggestion
- « Ouvrir *nom de l'app* »
```

File: src/fr/articles/how-to-test-android.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+- « Monter le volume »
+
+Il est possible d'afficher les libellés et les numéros des éléments interactifs. Aussi, pour qu'une application soit pilotable de la sorte, il faut que les composants interactifs aient un nom accessible simple et cohérent (en particulier dans le cas d'une image-lien ou image-bouton sans libellé visible).
```
Comment (MewenLeHo) [inline src/fr/articles/how-to-test-android.md:71]: les numéros ?

File: src/fr/articles/how-to-test-android.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+- l'état des éléments (coché ou non, sélectionné ou non, déplié ou non...),
+- les titres des pages doivent être restitués pour permettre aux utilisateurs de s'assurer d'un changement de page et d'en comprendre le contexte,
+- les changements de contenus ou messages temporaires doivent également être lus (alertes, erreurs, changement de contenu...),
+- l'ordre de lecture est cohérent par rapport à la présentation de l'information.
+
+
+### Navigation clavier
+Il est possible d'utiliser son téléphone seulement avec un clavier externe.
+1. Connecter un clavier au téléphone
+2. A présent, toutes les fonctionnalités disponibles par gestes tactiles doivent l’être également à l’aide du clavier.
```
Comment (MewenLeHo) [inline src/fr/articles/how-to-test-android.md:53]: Pas les bonnes apostrophes ;)

File: src/en/articles/how-to-test-android.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+2. Maximum font size corresponds to a 200% zoom.
+
+3. If the application has been designed to support enlargement (Dynamic Type), then the texts will be correctly rendered, readable, and without loss of information (no truncated text or superposition).
+
+   ![](https://github.com/user-attachments/assets/aaadf571-bb66-4455-9525-bd21df31bacc)  ![](https://github.com/user-attachments/assets/bac1f246-faeb-4cfc-b5df-4084c530d4e2)
+
+
+### Dark mode
+As the dark mode is used more and more, so it's strongly recommended to test your application by activating “dark mode”.
```
Comment (MewenLeHo) [inline src/en/articles/how-to-test-android.md:95]: 1. simples quotes only in english so 'dark mode'.
2. As the dark mode is used more and more, it's strongly recommended to test your application by activating 'dark mode' OR As the dark mode is used more and more, it's strongly recommended to test your application by activating 'dark mode'.

**Review by MewenLeHo — CHANGES_REQUESTED** [review]
Quelques suggestions :)

**Review by afrahB — CHANGES_REQUESTED** [review]
Replace the title "Testing the accessibility of an iOS application" with "Testing the accessibility of an Android application."

**Review by pya35 — COMMENTED** [review]
OK

File: src/fr/articles/how-to-test-android.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+Des combinaisons sont également spécifiques à l'environnement mobile Android :
+
+![Exemples de raccourcis : CMD+ENTREE pour l'accueil, CMD+SUPPR pour retour, ALT+TAB pour récents, ALT+F4 pour fermer l'application en cours, CMD+! pour accéder aux raccourcis clavier](https://github.com/user-attachments/assets/46a81d5a-5dcb-4232-a206-8baf39366eb0)
+
+
+3. La navigation clavier permet également de vérifier que l'ordre de navigation est respecté.
+
+
+### Voice Access
+Lorsque « Voice Access » est activé, vous pouvez énoncer des commandes comme :
```
Comment (afrahB) [inline src/fr/articles/how-to-test-android.md:65]: Ajouter la commande "Afficher les libellés" pour afficher le libellé correspondant à chaque élément cliquable

Comment (pya35) [reply src/fr/articles/how-to-test-android.md:65]: déjà fixé par ce commit https://github.com/Orange-OpenSource/a11y-guidelines/pull/545/commits/f446836375c58e3c913c6f5e7d23a6c96696c26c

**Review by pya35 — COMMENTED** [review]
OK

File: src/en/articles/how-to-test-android.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+
+### Voice Access
+When "Voice Access" is enabled, you can say commands like:
+- "Go home "
+- "Tap *item name*"
+- "Open *app name*"
+- "Turn volume up"
+
+By default, element names are superimposed.  Also, for an application to be controllable in this way, the interactive components must have a simple and consistent accessible name (in particular in the case of a link image or button image without a visible label).
```
[the remark this answers is not in the document]
Comment (pya35) [reply src/en/articles/how-to-test-android.md]: déjà fixé par ce commit https://github.com/Orange-OpenSource/a11y-guidelines/commit/f446836375c58e3c913c6f5e7d23a6c96696c26c

**Review by Aniort — APPROVED** [review]
relecture ok

**Comment by pya35** [comment]
@MewenLeHo @afrahB c'est OK pour vous ?

**Review by MewenLeHo — APPROVED** [review]
LGTM! 🚀

**Review by afrahB — APPROVED** [review]

**Pull request merged**